src/app/components/MnemonicGrid/index.tsx

```diff
@@ -39,7 +39,9 @@ function MnemonicWord(props: WordProp) {
         <Text style={noSelect}>{props.id}.</Text>
       </Box>
       <Box>
-        <strong style={keepWhitespace}>{props.hidden ? '' : props.word}</strong>
+        <strong className="notranslate" translate="no" style={keepWhitespace}>
```

Comment:
is there a place where we can put a reference to `notranslate`?

https://cloud.google.com/translate/troubleshooting

Reply:
+1

Refactored into https://github.com/oasisprotocol/oasis-wallet-web/pull/522/commits/9fb967c0dd66c65f8a9ebf194565b10a4d14541e#diff-332f02a3f0ce05164f077de08ce661dc404252bb792e63e899f03b0f9ad6ed91R7-R20